swagger-doc-extractor/build.gradle

```diff
@@ -11,6 +11,7 @@ ext {
 }
 
 dependencies {
+    implementation(project(':apix-rest-v1'))
```

Comment:
Why this dep?

Reply:
Since moving the documentwebscript class pulls in api classes. but I suppose the impl dep provides those transitively